packages/server/graphql/mutations/acceptTeamInvitation.ts

```diff
@@ -74,7 +74,7 @@ export default {
       const {meetingId, teamId} = invitation
       const meeting = meetingId ? await dataLoader.get('newMeetings').load(meetingId) : null
       const activeMeetingId = meeting && !meeting.endedAt ? meetingId : null
-      const team = await dataLoader.get('teams').load(teamId)
+      const team = (await dataLoader.get('teams').load(teamId))!
```

Comment:
+1 this is correct, but i sure wish there were another way we could do this.
could we override the `load` method coming from DataLoader with a generic so writing `load<true>(teamId)` returns a `Promise<Omit<V, null>>`?

true is just a dummy, we could use any value, even a string.